src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getReviews } from "../utils/api";
+
+function ReviewsList() {
+  const [reviewsList, setReviewsList] = useState([]);
+  useEffect(() => {
+    getReviews().then((reviewsList) => {
+      setReviewsList(reviewsList);
+    });
+  }, []);
+
```

Comment:
We want to use a loading state to conditionally render some sort of loading message whilst we retrieve information from the api